openedx/core/djangoapps/credit/models.py

```diff
@@ -55,11 +56,19 @@ def get_credit_course(cls, course_key):
 class CreditProvider(TimeStampedModel):
     """This model represents an institution that can grant credit for a course.
 
-    Each provider is identified by unique ID (e.g., 'ASU').
+    Each provider is identified by unique ID (e.g., 'ASU'). CreditProvider also
+    includes a `url` where the student will be redirected when he/she will try
```

Comment:
"redirected" might be confusing here, since we're not using an HTTP redirect.  Maybe say, "where the student will be sent..."